blog.jh

```diff
@@ -1,9 +1,24 @@
+application {
+  config {
+    baseName blog,
+    applicationType monolith,
+    packageName com.jhipster.demo.blog,
```

Comment:
I'd change this from `com.jhipster` to `org.jhipster` since I own that domain name. Or not... 

I just checked jhispter.com, and it looks like @jdubois might've bought that since it redirects to https://www.jhipster.tech/?refer=com.

Reply:
Actually we don't own Jhipster.com, we have no idea who owns it. So ya org.jhipster or tech.jhipster is better